src/app/pages/client/explore/Explore.tsx

```diff
@@ -199,67 +317,68 @@ export function Explore() {
               </NavLink>
             </NavItem>
             {userServer && (
-              <NavItem
-                variant="Background"
-                radii="400"
-                aria-selected={selectedServer === userServer}
-              >
-                <NavLink to={getExploreServerPath(userServer)}>
-                  <NavItemContent>
-                    <Box as="span" grow="Yes" alignItems="Center" gap="200">
-                      <Avatar size="200" radii="400">
-                        <Icon
-                          src={Icons.Server}
-                          size="100"
-                          filled={selectedServer === userServer}
-                        />
-                      </Avatar>
-                      <Box as="span" grow="Yes">
-                        <Text as="span" size="Inherit" truncate>
-                          {userServer}
-                        </Text>
-                      </Box>
-                    </Box>
-                  </NavItemContent>
-                </NavLink>
-              </NavItem>
+              <ExploreServerNavItem
+                server={userServer}
+                selected={userServer === selectedServer}
+                icon={Icons.Home}
+              />
+            )}
+            {exploringUnlistedServer && selectedServer !== undefined && (
+              <ExploreServerNavItem
+                server={selectedServer}
+                selected
+                icon={Icons.Server}
+                action={{
+                  alwaysVisible: true,
+                  icon: Icons.Plus,
+                  onClick: () => addServerCallback(selectedServer, true),
+                }}
+              />
```

Comment:
it should be rendered just above the "add server" button so it doesn't switch places on "Plus" "minus" click